coldfront/plugins/qumulo/tests/test_quotas.py

```diff
@@ -251,6 +652,16 @@ def setUp(self) -> None:
 
         return super().setUp()
 
+    def test_qumulo_result_set_page_limit_should_be_set(self) -> None:
+        page_limit = qumulo_api.QumuloAPI.get_result_set_page_limit()
+        self.assertIsNotNone(page_limit)
+
+    @unittest.skip("Until we have a chance to propagte the ENV variable.")
```

Comment:
In the meanwhile, until that exception is safe to throw, it might be worth a test to try different numbers of the page limit - like 10, and assert there are 10 results, and then 20, etc.  To confirm the setting is taking effect.

Reply:
This should be an integration test.

Reply:
I added an integration test to test the page_limit without mocking the Qumulo API call.